corehq/apps/app_manager/models.py

```diff
@@ -4114,7 +4114,7 @@ def wrap(cls, data):
         if "_attachments" in data:
             data = data.copy()
             attachments = data.pop("_attachments").copy()
-            if cls.migrating_blobs_from_couch:
+            if cls._migrating_blobs_from_couch:
```

Comment:
Very true. When I designed `jsonobject` I was both blessed with having couchdbkit's interface already designed for me to match exactly, and cursed with the blemishes of that design. woh, woh